### ISSUE TITLE
Consider renaming `lint-check` to `lint`

### ISSUE THREAD

**Issue by jfparadis** [Agoric/agoric-sdk#616]
Tooling currently exposes scripts `lint-check` and `lint-fix`. This corresponds to `eslint` and `eslint --fix`. Most repos use `lint` to mean the default, check only.

**Comment by warner** [comment]
Sounds good to me.. would we have `lint` and `lint-fix` then?

(I'm still a bit uncertain about what to do with `prettier` and `prettier-fix`)

**Comment by jfparadis** [comment]
Yes, `lint` and `lint-fix`. Remove the prettier script, let eslint handle prettier via the prettier plugin. On some cases the output of prettier can be different than eslint + prettier plugin.
